tests/Resources/Common.robot

```diff
@@ -151,5 +151,17 @@ Skip If RHODS Version Greater Or Equal Than
        Skip If    condition=${version-check}==True    msg=This test is skipped for RHODS ${version} or greater
     END
 
-
-
+Get Domain From Current URL
+    [Documentation]    Gets the lowest level domain from the current URL
```

Comment:
Can you add some examples in the [Documentation]?

Reply:
Done